constraints.txt

```diff
@@ -251,7 +252,7 @@ ruamel.yaml.clib==0.2.7
 scipy==1.11.1
 seaborn==0.12.2
 SecretStorage==3.3.3
-semsimian==0.1.20
+semsimian==0.1.18
```

Comment:
@hrshdhgd it seems semsimian got downgraded for some reason, while everything else (SSSOM, LinkML, etc) was nicely updated?

Reply:
I am more concerned by the downgrade of `pyparsing` from 3.0.9 down to 2.4.7 – there are known breaking changes between the 2.x and 3.x branches.

Reply:
`semsimian` downgrade makes sense. It is via `oaklib` and 0.1.18 works well with the last release if `oaklib` , so it is pinned and that's fine. There's active development going on with this so I wouldn't worry too much about this.

`pyparsing`: I have no clue.

Reply:
Ok thanks..